src/Db/db.rb

```diff
@@ -2,6 +2,7 @@
 require 'openssl'
 require 'open3'
 PKSIM_DB_FILE_NAME = 'PKSimDB.sqlite'
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
```

Comment:
After I've added environment variable **HTTPS_PROXY** and set it to my default windows proxy AND added this line, I am able to download the database via the rake script.

Reply:
>What does that mean
HTTPS_PROXY and set it to my default windows proxy

Is set to the http address of my windows proxy. Not including userid or password - so this will not be changed every month. If rake would give the possibility for something like 
`set HTTPS_PROXY=${DEFAULT_WINDOWS_PROXY}` there would be no need for definition of extra environment variables.

>I had to add this line for the suite setup at some point. Maybe it's enough with it without the HTTPS_PROXY?

Nope, download still fails if the proxy variable is not set.